### PR TITLE
when `doing open issue` is invoked without a parameter then open the `current issue` i.e. based on the checked out branch name

### PR DESCRIPTION
**Pull request by FitzgeraldKrudde**
Be aware this is my first python commit ever :-)

I tried to keep inline with the current code/style

### CODE REVIEW

File: src/doing/utils.py [hunk 1]
```diff
@@ -255,3 +255,14 @@ def replace_user_aliases(text: str) -> str:
         return " ".join(words)
     else:
         return " ".join([aliases.get(word, word) for word in words])
+
+def get_current_work_item_id():
+    """
+    Retrieves current work item id from the current branchname
+    """
+    wi_id = shell_output("git branch --show-current | cut -d '_' -f 1 | sed 's/[^0-9]*//g'")
```
Comment (timvink) [inline src/doing/utils.py:263]: `sed` has some subtle differences between OS and shells.

I'd prefer to do as much as possible in python. So something like:

```python
import re 
branch_name =  shell_output("git branch --show-current")
wi_id = re.search(r"^([0-9]*)_", branch_name).group(1)
..
```

If you're not comfortable doing this in python, let me know & I'll pick it up.

File: src/doing/open/commands.py [hunk 2]
```diff
@@ -94,16 +94,17 @@ def pipe():
 
 
 @open.command()
-@click.argument("work_item_id")
+@click.argument("work_item_id", default=-1)
 def issue(work_item_id):
     """
-    Open a specific WORK_ITEM_ID.
+    Open a specific WORK_ITEM_ID or when not provided open the workitem based on the branchname.
```
Comment (timvink) [inline src/doing/open/commands.py:100]: First sentence needs to be short (it's what displays when you type `doing open`)

So f.e.:

```
Open a specific WORK_ITEM_ID.

When not provided attempt to auto-detect the WORK_ITEM_ID using the git branch name
```

**Review by timvink — CHANGES_REQUESTED** [review]
Thanks for the contribution & nice idea! Not bad for your first python PR :) And fits nicely with a similar `doing open pr` (#33).

Some small comments. Also, can you update `docs/reference/manual/open.md` with example usage?

File: src/doing/utils.py (hunk 1 above)
Comment (FitzgeraldKrudde) [reply src/doing/utils.py]: implemented and tested your suggestion, works flawless!

File: src/doing/open/commands.py (hunk 2 above)
Comment (FitzgeraldKrudde) [reply src/doing/open/commands.py]: rephrased

**Comment by timvink** [comment]
Thanks for the work on this! Will release shortly.

**Comment by timvink** [comment]
Released https://github.com/ing-bank/doing-cli/releases/tag/v1.1.8 !

**Pull request merged**